flax/linen/module.py

```diff
@@ -411,23 +463,28 @@ def wrapped_module_method(*args, **kwargs):
       return self._call_wrapped_method(fun, args, kwargs)
     else:
       return fun(*args, **kwargs)
+
   wrapped_module_method.method_handler_wrapped = True  # type: ignore[attr-defined]
   return wrapped_module_method
 
 
 def _wrap_hash(hash_fn: Callable[..., Any]) -> Callable[..., Any]:
   """Wraps a hash function with some check for Flax Modules."""
+
   @functools.wraps(hash_fn)
   def wrapped(self):
     if self.scope is not None:
-      raise TypeError('Can\'t call __hash__ on modules that hold variables.')
+      raise TypeError("Can't call __hash__ on modules that hold variables.")
```

Comment:
It is a bit odd these contain double quotes while the rest of the changes in this file seem to be single quotes.

Reply:
the black style is to avoid `\` whenever possible, in this case by switching the outer quote style